.eslintrc.js

```diff
@@ -5,19 +5,13 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: 'module',
     ecmaFeatures: {
-      legacyDecorators: true
-    }
+      legacyDecorators: true,
+    },
   },
-  plugins: [
-    'ember',
-    'hbs'
-  ],
-  extends: [
-    'eslint:recommended',
-    'plugin:ember/recommended'
-  ],
+  plugins: ['ember', 'hbs'],
```

Comment:
Shouldn't `prettier` be also added as a plugin here? Not really sure though, but that's what we have in the common setup of our apps, and Ember seems to have it also: https://github.com/emberjs/ember.js/blob/master/.eslintrc.js#L14

Reply:
I tried to follow the recommended configuration for ESLint integration in Prettier docs: https://prettier.io/docs/en/integrating-with-linters.html#recommended-configuration

Reply:
But doesn't the section above it say you should add prettier to the `plugins` array: https://prettier.io/docs/en/integrating-with-linters.html#use-eslint-to-run-prettier. Or is that happening implicitly when using `prettier/recommended`? The docs seem to be not very clear, at least for me...

Reply:
The recommended configuration combines both steps mentioned before. At least if I got it right. The config is working fine for a bunch of apps and addons since a few months.

Reply:
Ok, convinced! :)